boards/unphone8.json

```diff
@@ -0,0 +1,41 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DUNPHONE_SPIN=8",
+      "-DARDUINO_USB_CDC_ON_BOOT=0",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1"
```

Comment:
There is `-DBOARD_HAS_PSRAM` in the PR to the Arduino core. Shouldn't it be added here as well for both `unphone8` and `unphone9`?